docs/library/testing.md

```diff
@@ -43,6 +53,32 @@ class TestableModel(Model[ModelItemType, ModelItemType]):
 
 ```
 
+#### Adding test_cases to the CONFIGURATIONS map
+
+Tests added to the CONFIGURATIONS map are restricted to their parent.
+
+In the following example, 2 test cases will be ran for `some_model_a`:
+
+```python
+class TestableModel(Model[ModelItemType, ModelItemType]):
+    CONFIGURATIONS: Dict[str, Dict] = {
+        "some_model_a": { 
+            "test_cases": {
+                "cases": [
```

Comment:
The same comment applies here, we should have `"test_cases" : [...]` instead of `"test_cases": {"cases": [...]}`